nmrglue/process/proc_autophase.py

```diff
@@ -14,46 +14,46 @@
 from .proc_base import ps
 
 
-def autops(data, fn, p0=0.0, p1=0.0):
+def psacme(data, p0=0.0, p1=0.0, gamma=5.0e-3):
```

Comment:
I do not think the name of this function should be changed.  If it is the name in the documentation must also be changed.